src/validation.cpp

```diff
@@ -4302,6 +4302,17 @@ static bool ContextualCheckBlock(const CBlock& block, BlockValidationState& stat
         }
     }
 
+    // Make sure the coinbase transaction is timelocked to the block's height.
+    if (nHeight > 0 && DeploymentActiveAfter(pindexPrev, chainman, Consensus::DEPLOYMENT_CONSENSUSCLEANUP)) {
```

Comment:
00745a2a82de926027b92871870a3929d19b7cc7

nit: nHeight==0 should never happen I think?